docs/pages/x/api/data-grid/data-grid.json

```diff
@@ -459,8 +459,8 @@
   "imports": [
     "import { DataGrid } from '@mui/x-data-grid/DataGrid';",
     "import { DataGrid } from '@mui/x-data-grid';",
-    "import { DataGrid } from '@mui/x-data-grid-pro';",
-    "import { DataGrid } from '@mui/x-data-grid-premium';"
+    "import { DataGridPro } from '@mui/x-data-grid-pro';",
+    "import { DataGridPremium } from '@mui/x-data-grid-premium';"
```

Comment:
The CI isn't happy because this is [autogenerated from elsewhere](https://github.com/mui/mui-x/blob/06e1e7284b7a97b7750c50ce613d7a334cf71c10/docs/scripts/api/buildInterfacesDocumentation.ts#L157). You can run the autogen code with `pnpm run docs:api`. I've never touched that code yet so I couldn't say why it's picking up the `DataGrid` name.

Reply:
ah. Sorry I can't dig in any further at this point. Thank you for reviewing @romgrk ! I'll go ahead and close for now.